### ISSUE TITLE
ctrlspace#api#projectroot()

### ISSUE THREAD

**Issue by Konfekt** [szw/vim-ctrlspace#151]
Now that a couple of functions, such as `ctrlspace#api#Statusline()` went public, how about making the function that gives the current project root public?

**Comment by szw** [comment]
Well, there is already `ctrlspace#roots#CurrentProjectRoot()`. I know it's not strictly in the `api` namespace, but after most functions have been declared public I consider if the API namespace is relevant at all.

**Comment by Konfekt** [comment]
Regarding relevancy: By uniform labels and documenting those, trust that these functions, in particular their names, are there to stay. 

Apparently `ctrlspace#roots#CurrentProjectRoot()` is only set after `:CtrlSpace`. Could the project root be read off even before ?

**Comment by szw** [comment]
No, in the current implementation it's not set until CtrlSpace is shown. Perhaps there could be an API function as you suggested that would perform this initialization before returning the current project root.

**Comment by Konfekt** [comment]
Ok, you may think about it. Currently there are plugins like `https://github.com/dbakker/vim-projectroot` that provide it, but for someone who uses CtrlSpace, it seems to be already essentially there.
